### PR TITLE
fix: Disabling the addition of one extra day in EndAtDate

### LINKED ISSUE #3020: endsAtDate attribute defined in Event Models is buggy
**Describe the bug**

My event start and end date were June 27,2019 and June 30,2019 respectively
Data stored in startsAtDate and endsAtDate were 6/27/2019 and 07/01/2019 respectively

**To Reproduce**

Steps to reproduce the behaviour:

Check the `event.endsAtDate` it would give a result greater than one day .

**Expected behaviour**

Find the cause of the bug and Fix it.

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3020 

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 @ritikamotwani Please Review

File: app/utils/computed-helpers.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ export const computedSegmentedLink = function(property) {
 export const computedDateTimeSplit = function(property, segmentFormat) {
   return computed(property, {
     get() {
-      if (property === 'endsAt' && segmentFormat === 'date') {
-        return moment(this.get(property)).add(1, 'days').format(getFormat(segmentFormat));
-      }
+      // if (property === 'endsAt' && segmentFormat === 'date') {
+      //   return moment(this.get(property)).add(1, 'days').format(getFormat(segmentFormat));
+      // } The following line was adding one extra day to the endsAt Attribute . Commenting it to fix the issue .
```
Comment (uds5501) [inline app/utils/computed-helpers.js]: @CosmicCoder96 This fixes the issue but I want to know why was this placed in the first place? Does it has any other utility which maybe undermined by removing this?

**Comment by kushthedude** [comment]
@uds5501 +1 ?

**Comment by kushthedude** [comment]
But i dont see any logic adding one day extra in end date of event , maybe
it could have something related to salesEndAt , but i am not sure 🤔

**Comment by kushthedude** [comment]
@CosmicCoder96 Please Review

File: app/utils/computed-helpers.js (hunk 1 above)
Comment (CosmicCoder96) [reply app/utils/computed-helpers.js]: Not sure. Though storing the wrong date is definitely wrong. Let's see in the test run, this week after merge.

**Pull request merged**